data/data/gcp/network/network.tf

```diff
@@ -37,7 +37,10 @@ resource "google_compute_router_nat" "master_nat" {
   nat_ip_allocate_option             = "AUTO_ONLY"
   min_ports_per_vm                   = 7168
   source_subnetwork_ip_ranges_to_nat = "LIST_OF_SUBNETWORKS"
-
+  log_config {
+    enable = true
+    filter = ERRORS_ONLY
+  }
```

Comment:
we don't enable logging for various other networking components and I don't think we should enable these for all the customers. the users should be able to enable this for their clusters whenever they want.